docs/experimenter/README.md

```diff
@@ -471,7 +470,7 @@ A live rollout can have updates pushed to its state while remaining Live. These
     rect rgb(255,204,255) 
         Note right of Experiment Owner: Owner makes changes in Experimenter
         Experiment Owner->>Experimenter UI: Make updates to live rollout
-        Experimenter UI->>Experimenter Backend: Status: Live <br/> Publish status: Dirty <br/> Status next: <none> <br/> + changelog
+        Experimenter UI->>Experimenter Backend: Status: Live <br/> Publish status: Idle <br/> Status next: <none> <br/> is_dirty: True <br/> + changelog
```

Comment:
This shows the UI setting the `is_dirty` flag, but why would we change this from the serializer setting it?  We shouldn't need to involve the frontend in it at all?

Reply:
Good catch. This is one of the things that is changing with the flag vs. publishstatus. I'll update these..